llama-index-integrations/llms/llama-index-llms-oci-genai/llama_index/llms/oci_genai/base.py

```diff
@@ -219,6 +190,7 @@ def _get_all_kwargs(self, **kwargs: Any) -> Dict[str, Any]:
             **kwargs,
         }
 
+    @deprecated("Deprecated in favor of `chat`, which should be used instead.")
```

Comment:
technically, you can make complete call chat -- we have utils to make this very easy. A few llms do this

Reply:
Thank you @logan-markewich Agreed. But basically all the models that support completion have been deprecated in our service. Let me take a look at other vendors.